velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -268,6 +268,21 @@ TEST_F(ArithmeticTest, cosh) {
   EXPECT_TRUE(std::isnan(cosh(kNan).value_or(0)));
 }
 
+TEST_F(ArithmeticTest, unhex) {
+  const auto unhex = [&](std::optional<std::string> a) {
+    return evaluateOnce<std::string>("unhex(c0)", a);
+  };
+
+  EXPECT_EQ(unhex("737472696E67"), "string");
+  EXPECT_EQ(unhex(""), "");
+  std::string b("#\0", 2);
+  EXPECT_EQ(unhex("123"), b);
```

Comment:
learned usage from https://stackoverflow.com/questions/164168/how-do-you-construct-a-stdstring-with-an-embedded-null